### PR TITLE
RAG Dev Bootcamp - add agenda to the README

### PR DESCRIPTION
**Pull request by marwan116**
adds an agenda to the README

### CODE REVIEW

**Comment by marwan116** [comment]
@kamil-kaczmarek do you think we explicitly add the Deployment bonus in the agenda ? or keep it less official ?

There is a bit of a worry that serve deploy might fail again given the root cause of the failure in dry run is still not figured out

File: templates/rag-dev-bootcamp-mar2024/README.md [hunk 1]
```diff
@@ -26,4 +26,17 @@ wget https://anyscale-materials.s3.us-west-2.amazonaws.com/rag-mar-2024.zip
 unzip rag-mar-2024.zip
 ```
 
-2. Navigate to the `01_overview` folder and open the `notebook.ipynb` file to get started with the training program.
\ No newline at end of file
+2. Navigate to the `01_overview` folder and open the `notebook.ipynb` file to get started with the training program.
+
+## Agenda
```
Comment (kamil-kaczmarek) [inline templates/rag-dev-bootcamp-mar2024/README.md:31]: 10:00 am -> 10:20 am - Set up 
10:20 am -> 10:40 am - RAG overview
10:40 am -> 11:10 am - Canopy
11:10 am -> 11:20 am - Coffee break
11:20 am  -> 12:00 pm - Embeddings pipeline Part 1
12:00 pm -> 1:00 pm - Lunch Break
1:00 pm -> 2:00 pm - Embeddings pipeline Part 2
2:00 pm  -> 3:00 pm - Evaluations Part 1
3:00 pm -> 3:15 pm - Coffee break
3:15 pm -> 4:00 pm - Evaluations Part 2
4:00 pm -> 4:30 pm - Bonus session (stay tuned)


What do you think? Added short break after session 1

Comment (kamil-kaczmarek) [reply templates/rag-dev-bootcamp-mar2024/README.md:31]: also change name of the last session.

Comment (marwan116) [reply templates/rag-dev-bootcamp-mar2024/README.md:31]: > What do you think? Added short break after session 1

I am ok with it given it signals a mindset switch from canopy to ray.

Just its length is 10 min, whereas the second coffee break is 15 min. Should we stick to 10 min for both ?

Comment (marwan116) [reply templates/rag-dev-bootcamp-mar2024/README.md:31]: > 4:00 pm -> 4:30 pm - Bonus session (stay tuned)
I like that it gives us leeway. Is the idea here for instance that we de-risk the service deployments by me trying to run the notebook in the last coffee break and make sure it works. If it doesn't then perhaps we cancel showcasing the deployment ?

**Review by kamil-kaczmarek — APPROVED** [review]

**Pull request merged**